README.md

```diff
@@ -48,20 +60,25 @@ You will need Visual Studio 2017 or later, and [git](https://git-scm.com).
 
 3. If Windows asks which program to open `spt.sln` with, choose Visual Studio.
 
+   To avoid retargeting, install `C++ Windows XP Support for VS 2017 (v141) tools [Deprecated]`.
```

Comment:
I think retargeting is due to Windows SDK updates, not due to the Windows XP support.

Also it's weird to say "to avoid retargeting do x" and then in the very next sentence "if it asks to retarget press ok".